content/docs/rfcs/14/README.md

```diff
@@ -34,7 +34,10 @@ To enable a bidirectional bridge with Waku v1 see [15/WAKU2-BRIDGE](/spec/15) fo
 The `version` field MAY be filled out to allow for various types of payload encryption.
 Omitting it means the version is 0.
 
-The `timestamp` field MAY be filled out to signify the time at which the message is generated by its sender. This field holds the Unix epoch time. Omitting it means the timestamp is unspecified.
+The `timestamp` field MAY be filled out to signify the time at which the message is generated by its sender. 
+This field holds the Unix epoch time in seconds. 
+The fractional part represents the sub-seconds. 
```

Comment:
I suppose this will be tweaked once timestamp is int based?

Reply:
That is right!